praw/models/reddit/subreddit.py

```diff
@@ -198,6 +199,13 @@ def moderator(self):
         return self._moderator
 
     @property
+    def modmail(self):
+        """An instance of :class:`.Modmail`."""
```

Comment:
Sphinx doesn't like this class reference, and I can't figure out why. It seems no different from the one in the docstring for `Subreddit.stream`.